packages/core/strapi/bin/strapi.js

```diff
@@ -277,7 +277,12 @@ program
       .argParser(parseInputBool)
   )
   .addOption(
-    new Option('--compress [boolean]', 'Compress output file using gz')
+    new Option('--compress [boolean]', 'Compress output file using gzip compression')
+      .default(true)
+      .argParser(parseInputBool)
+  )
+  .addOption(
+    new Option('--archive [boolean]', 'Archive output files using tar')
```

Comment:
`Archive output files using tar` Does this mean it creates an archive for every archive file? :thinking:

Reply:
I know, I should work on my English. Fixed 😆

Though it almost makes sense with our discussion on compressing/encrypting individual files...